src/shared/lib/comparison/ComparisonStore.ts

```diff
@@ -1078,7 +1078,7 @@ export default abstract class ComparisonStore extends AnalysisStore
             if (
                 (this.alterationsEnrichmentDataRequestGroups.result &&
                     this.alterationsEnrichmentDataRequestGroups.result.length >
```

Comment:
Maybe simplify it this way?
```suggestion
                this.alterationsEnrichmentDataRequestGroups.result?.length
```
Technically, we don't even need the check `> 0`.

Reply:
Personally I prefer to have the > 0 since it's more explicit, but I'm happy to change it if you want me to.

Reply:
We can keep the `> 0`. I don't have a strong preference.

Reply:
Ah, just realized if we keep the `> 0` then the above simplification doesn't work. TypeScript complains if we do

```JavaScript
this.alterationsEnrichmentDataRequestGroups.result?.length > 0
```

Reply:
Sorry for dragging this out but if we really want to be explicit we can also do this:

```JavaScript
!_.isEmpty(this.alterationsEnrichmentDataRequestGroups.result)
```